docs: clarify how the binary works (#568)
containerd/runwasi: README.md

sudo make install
```

> > Note: `make build` will only build one binary. The `make install` command copies the binary to $PATH three times with different names for each component described above. 

Reviewer: It's actually 1 binary and 2 symlinks, and not the same binary copied 3 times.
```suggestion
> Note: `make build` will only build one binary. The `make install` command copies the binary to $PATH and uses symlinks to create all the component described above. 
```
